test/TestSites/web.config

```diff
@@ -6,6 +6,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\TestSites.exe" arguments="" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true" />
+    <aspNetCore stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true" />
```

Comment:
The problem is that if you have a standalone app, the publish-iis tool only overwrite the process path and leaves the envvar in the arguments attribute. Better to remove both and let the publish-iis tool set them if needed.